packages/shared/src/interface/external/surescripts/payment-code.ts

```diff
@@ -22,3 +26,56 @@ export const PaymentCodeName: Record<PaymentCode, PaymentCodeName> = {
   "07": "Indian Nations",
   "99": "Other",
 };
+
+export function getSourceOfPaymentCode(paymentCode: string): SourceOfPaymentCode | undefined {
+  return PaymentCodeToSourceOfPaymentCode[paymentCode as PaymentCode];
+}
+
+export function getSourceOfPaymentName(sourceOfPaymentCode: string): string | undefined {
+  return SourceOfPaymentName[sourceOfPaymentCode as SourceOfPaymentCode];
+}
+
+// https://www.nahdo.org/sopt
+const SourceOfPaymentCodes = [
+  "1",
+  "2",
+  "3",
+  "312",
+  "33",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "89",
+  "95",
+  "99",
+] as const;
+export type SourceOfPaymentCode = (typeof SourceOfPaymentCodes)[number];
+
+export const SourceOfPaymentName: Record<SourceOfPaymentCode, string> = {
```

Comment:
nit: wouldn't it be better to sort by value/number instead of lexicographically?